airflow/contrib/operators/dataproc_operator.py

```diff
@@ -716,7 +721,10 @@ def execute(self, context):
         job.add_jar_file_uris(self.dataproc_jars)
         job.set_job_name(self.job_name)
 
-        hook.submit(hook.project_id, job.build(), self.region)
+        job_to_submit = job.build()
+        self.dataproc_job_id = job_to_submit["job"]["reference"]["jobId"]
```

Comment:
This is understandable, and you can use this to create new Operators or functions in the existing operators.